### ISSUE TITLE
Improve react native example component

### ISSUE THREAD

**Issue by IskandarovRost** [cuba-platform/frontend#231]
see: https://doc.cuba-platform.com/frontend/cuba-frontend-docs/v3.0.x/client-react-native/starter-guide.html

1) Create some project on platform 7.2
2) It is required to add **REST API app component**
![image.png](https://images.zenhubusercontent.com/5aec53f14b5806bc2bd1c28a/0281bfe8-e304-4727-bebf-2ccf326cf3a3)
3) Generate react native client
``gen-cuba-front react-native:app --model /home/rost/StudioProjects/testrn/projectModel.json -d ./rnclient``
4) ``npm install``
5) ``expo start -web``

Then:
6) Login to middelware
7) Create security role with the Rest- scope
8) enable Rest access in this role (Specific permission)
![image.png](https://images.zenhubusercontent.com/5aec53f14b5806bc2bd1c28a/6b1e981d-96b5-4ada-bbb3-74ed545e0033)
9) and add this role to user

10) then login to frontend by this user

AR: endless progressbar. 
It seems, _the list of all users_ is not great for the example, as the user should have extra permissions to read the User entity via REST. 
![image.png](https://images.zenhubusercontent.com/5aec53f14b5806bc2bd1c28a/ea6d4e66-4d31-4668-bab4-048753eec7ee)

**Comment by web-devel** [comment]
Agree, see cuba-platform/restapi/issues/105 and cuba-platform/frontend/issues/233 .
Users were picked only to demonstrate data loading (as built-in entities which definitely exist).
